web/src/app/webview/src/configuration.ts

```diff
@@ -27,6 +28,13 @@ export class WebviewConfiguration extends EngineConfiguration {
 
     return baseReport;
   }
+
+  onRuleFinalization(ruleBehavior: RuleBehavior) {
+    if(!isEmptyTransform(ruleBehavior.transcription?.transform)) {
+      const transform = ruleBehavior.transcription.transform;
+      this.oninserttext(transform.deleteLeft, transform.insert, transform.deleteRight);
```

Comment:
```suggestion
      this.oninserttext(transform.deleteLeft, transform.insert, transform.deleteRight ?? 0);
```

Not sure if this is required but given another place where it was done?